win32ss/gdi/gdi32/objects/text.c

```diff
@@ -500,6 +500,10 @@ ExtTextOutW(
                   cwc,
                   lpDx);
 
+    if ( !(fuOptions &  (ETO_GLYPH_INDEX | ETO_IGNORELANGUAGE)))
```

Comment:
Extra space before '!' and after '&'

Reply:
`if (!(fuOptions &(ETO_GLYPH_INDEX | ETO_IGNORELANGUAGE)))` like this?